main.py

```diff
@@ -140,6 +140,19 @@ def change_wallpaper(db, arg):
             print_columns(suggestions[1:])
             scripter.change_wallpaper(suggestions[0])
 
+def multiple_argument_handler(arg, time):
```

Comment:
What if the user inputs a string for time?
```pokemon slideshow hax```
What if the user inputs a negative number for time?
```pokemon slideshow -2```
Make sure to check for these edge cases and let the user know that they did something wrong instead of letting the program crash.